### PR TITLE
fix: rax-scrollview and rax-recyclerview support touch events in miniapp

### PR DESCRIPTION
**Pull request by shiftj18**
Since 1.15.0, scroll-view has support [touch events](https://docs.alipay.com/mini/component/scroll-view#%E5%B1%9E%E6%80%A7)  in native MiniApp. But, touch events at rax-scrollview and rax-recyclerview is invalid.

### CODE REVIEW

File: packages/rax-scrollview/src/miniapp/index.ts [hunk 1]
```diff
@@ -61,5 +65,53 @@ Component({
         scroll_animation_duration: duration,
       });
     },
+    onTouchStart: function onTouchStart(e) {
+      const event = fmtEvent(this.props, e);
+      event.nativeEvent = {
```
Comment (SoloJiang) [inline packages/rax-scrollview/src/miniapp/index.ts]: 为啥要加 `nativeEvent` ？

Comment (SoloJiang) [reply packages/rax-scrollview/src/miniapp/index.ts]: 文档顺便加一下吧

Comment (shiftj18) [reply packages/rax-scrollview/src/miniapp/index.ts]: 小程序原生事件的target/currentTarget内容跟fmtEvent后的target/currentTarget不完全一致，比如currentTarget中的offsetTop/offsetLeft；所以放到nativeEvent中备用。

Comment (shiftj18) [reply packages/rax-scrollview/src/miniapp/index.ts]: Api文档添加上了。

**Pull request merged**